maintainers/maintainer-list.nix

```diff
@@ -21234,6 +21234,11 @@
     githubId = 80012;
     name = "Simon Jagoe";
   };
+  sjamaan = {
```

Comment:
This needs to be a separate commit.

Reply:
Done!

Unfortunately, splitting the commit caused the author info of the original commit to be lost.